src/main/java/seedu/address/model/util/SampleDoctorDataUtil.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.util;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Doctor;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Contains utility methods for populating {@code AddressBook} with sample data.
+ */
+public class SampleDoctorDataUtil {
+    public static Doctor[] getSampleDoctors() {
+        return new Doctor[] {
+            // Note: putting space in Tags will result in invalid Tag name exception
+            new Doctor(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+                        new Address("Blk 30 Geylang Street 29, #06-40"),
+                        getTagSet("physician")),
+            new Doctor(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
+                        new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
+                        getTagSet("physician", "medicine")),
+            new Doctor(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
+                        new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
+                        getTagSet("psychologist")),
+            new Doctor(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
+                        new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
+                        getTagSet("general")),
+            new Doctor(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
+                        new Address("Blk 47 Tampines Street 20, #17-35"),
+                        getTagSet("internal")),
+            new Doctor(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
+                        new Address("Blk 45 Aljunied Street 85, #11-31"),
+                        getTagSet("heart"))
+        };
+    }
+
+    public static ReadOnlyAddressBook<Doctor> getSampleDoctorRecords() {
+        AddressBook<Doctor> sampleDoctorRecords = new AddressBook<>();
+        for (Doctor samplePerson : getSampleDoctors()) {
+            sampleDoctorRecords.addPerson(samplePerson);
+        }
+        return sampleDoctorRecords;
+    }
+    /**
+     * Returns a tag set containing the list of strings given.
+     */
+    public static Set<Tag> getTagSet(String... strings) {
+        return Arrays.stream(strings)
+                .map(Tag::new)
+                .collect(Collectors.toSet());
+    }
+}
+
+
```

Comment:
There was no need to create a separate class. The methods could fit under SampleDataUtil.